bothan-api/server/src/api/utils.rs

```diff
@@ -8,14 +7,13 @@ use crate::proto::price::{Price, Status};
 
 pub fn parse_price_state(id: String, price_state: PriceState) -> Price {
```

Comment:
Please add test of this case to prevent we check all case of price state

Reply:
added unit tests